src/oaklib/converters/obo_graph_to_rdf_owl_converter.py

```diff
@@ -89,6 +90,14 @@ def _convert_node(self, source: Node, target: rdflib.Graph) -> rdflib.Graph:
         uri = self._uri_ref(source.id)
         if not source.type or source.type == "CLASS":
             target.add((uri, RDF.type, OWL.Class))
+        if source.type == "PROPERTY":
+            if source.property_type == PropertyType.OBJECT:
+                target.add((uri, RDF.type, OWL.ObjectProperty))
+            elif source.property_type == PropertyType.ANNOTATION:
+                target.add((uri, RDF.type, OWL.AnnotationProperty))
+            elif source.property_type == PropertyType.DATATYPE:
+                target.add((uri, RDF.type, OWL.DatatypeProperty))
+            target.add((uri, RDF.type, OWL.Class))
```

Comment:
Is this row necessary in this if clause? Seems out of place

Reply:
thx, fixed